__tests__/app.test.js

```diff
@@ -49,3 +49,41 @@ describe('GET /api', () => {
         });
     });
   });
+
+  describe('GET /api/articles/:article_id', () => {
+    test('GET: 200 - returns an article object', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then(({ body }) => {
+                expect(body.articles).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    body: expect.any(String),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
+          });
+      });
+    });
+
+    test('GET: 404 - returns "Not Found" for non-existent article_id', () => {
+        return request(app)
+            .get('/api/articles/9999')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Not Found');
```

Comment:
error message could be more specific "article does not exist" instead of "Not found"